concentratetimer/concentratetimer.py

```diff
@@ -55,12 +57,35 @@ def create_widgets(self):
         self.total_clock_aim.config(text=f"Aim: {self.data.aim_clock_count}")
         self.total_clock_counts = tk.Label(self, height=1, width=15, textvariable="")
         self.total_clock_counts.config(text=f"Done: {self.data.total_clock_count}")
+
+        # self.prompt_label = tk.Label(self,
+        #                              text="Goal for this clock:",
+        #                              height=1,
+        #                              width=15,
+        #                              textvariable="")
+        # self.prompt_entry = tk.Entry(self)
```

Comment:
We are familiar with git/version control tools now. We could safely remove any useless code. It will be a good development discipline.